backends-common/rabbitmq/src/test/java/org/apache/james/backends/rabbitmq/RabbitMQExtension.java

```diff
@@ -144,7 +144,7 @@ public void beforeEach(ExtensionContext extensionContext) throws Exception {
             ReactorRabbitMQChannelPool.Configuration.builder()
                 .retries(2)
                 .minBorrowDelay(Duration.ofMillis(5))
-                .maxChannel(3));
+                .maxChannel(10));
```

Comment:
why?

Reply:
Overwize Distributed Task manager tests are unstable. Increasing this solves the issue. Don't ask me why.

Reply:
Ahahah so we change an arbitrary number to another one without letting even a comment because it solves CI issues by accident. What could go wrong?